### DISCUSSION TITLE
Tell me why the bottom panel is missing in the program. Is the program installed on Windows 10 64 bit?

### DISCUSSION THREAD

**Discussion by LLITUK** [arkypita/LaserGRBL#1808] [Q&A] [answered]
![38702947-7a3ce0fe-3e91-11e8-98ac-765ddb4e78960](https://user-images.githubusercontent.com/6561389/162069240-f849d55f-7498-4e1c-a9f1-e2028345ae4d.jpg)

**Comment by arkypita** [comment, marked as answer]
From LaserGRBL documentation: https://lasergrbl.com/usage/overrides/

![image](https://user-images.githubusercontent.com/8782035/162072653-ab97feb8-5dbe-4582-92a7-0a0599ef19d7.png)

**Reply by LLITUK** [reply]
Thank you very much for your reply. Your answer helped solve the problem.
